**Review of: A guide to sequence your favorite plant genomes**

Peter Tran

Li and Harkess present a whirlwind tour on how to sequence a plant genome in the era of modern genomics. To start, the authors offer some considerations regarding the plant in question as to what is feasible to sequence and what techniques to determine their feasibility. Since plant genomes vary greatly, the authors suggest finding closely related plants that have small haploid or diploid genomes to sequence (as polyploid genomes are difficult to assemble). To measure the size of a genome, the authors suggest flow cytometry; k-mer frequency plots can be generated to estimate heterozygosity levels, as heterozygosity increases assembly complexity; and then there needs to be an adequate amount of DNA isolated that is pure enough to not inhibit sequencing protocols.

Once considerations are dealt with, the first choice researchers will come up against is deciding what sequencing platform to use. Sequencers are broken up into ones that generate short, cheap, and accurate reads and ones that generate long, expensive, and error-prone reads. Illumina is commonly used for short-read sequencing and PacBio or Oxford Nanopore for long-read. Nanopore has the added benefit of being extremely small and portable.

After sequencing is performed and data is obtained, researchers will need to assemble the genomes. This is where choices made in sequencer play a large role. There are two main approaches to assembly, de Bruijn graph (DBG) or Overlap-Layout-Consensus (OLC). For long-read sequences, OLC works best; and short-read sequences, DBG works best. An issue with long-read sequencers is that high coverage of a large genome is difficult. The authors recommend a hybrid approach by generating short-read data, assembling the short reads into longer reads with DBG, then assembling those larger reads with OLC.

The final step in assembly is to combine contigs into scaffolds. This requires long-range information that can be acquired in multiple ways. A mate-pair library can be prepared that will hold >1 kbp of range information, though the authors mention that long-read sequencing exceeds 10-30 kbp in range. Hi-C can be used to capture long-range information of proximal DNA in histones. BioNano optical mapping can be used to create and visualize nicks in the genome, creating a genomic restriction map. 10x Genomics offers a microfluidic controller that is able to barcode DNA molecules which can then be read in the sequencing output. Once assembly is complete, researchers will need to validate their genome by annotating it.

In this paper, Li and Harkess gave a comprehensive walkthrough of a genome sequencing project. Most of their review appears to be non-specific to plants and could be applied to any organism, but the plant-specific considerations, in the beginning, are important.