**Review of: A survey of best practices for RNA-seq data analysis**

Peter Tran

This paper my Mortazavi _et al._ is _the paper_ when it comes to how to do RNA-seq analysis. The authors begin by giving a background introduction to RNA-seq, with the main highlights being that RNA-seq has been widely adopted in biological research, that there is no one optimal pipeline for analysis, and so the authors aim to outline the current standards and provide a guideline.

Before any analysis can occur, special consideration must be given to the experimental design. rRNA must be removed to leave the mRNA for analysis; polyA selection can enrich mRNA or rRNA can be degraded. In bacteria, polyA selection will not work. Sequencing can be done as single-end or paired-end, with paired-end being more expensive but easier for _de novo_ transcript discovery and isoform expression analysis; they are also easier for mapping. Sequencing depth must also be considered, with a higher depth comes more precision; saturation curves can be used to determine what depth of sequencing should be used. Lastly, a high number of replicates should be used to increase statistical power.

Once data is generated, analysis can occur. The first step is to perform some quality control, this will need to be done after obtaining raw reads, read alignment, and quantification. After receiving raw reads, `FastQC` and `NGSQC` can be used to check sequence quality, GC content, presence of adaptors, overrepresented k-mers and duplicated reads. Read quality decreases towards the 3' end, `Trimmomatic` and `FASTX-Toolkit` can be used to trim those reads. After aligning reads to a genome or transcriptome, 70-90% of reads should map to a human genome. There should also be uniformity of read coverage on exons and the mapped strand, reads accumulating in the 3' end of transcripts in polyA-selected samples may be of low quality. `Picard`, `RSeQC`, and `Qualimap` can be used to quality control mapping. Once transcripts are quantified, the GC content and gene lengths should be checked for bias. `NOISeq` and `EDASeq` can be used to do this.

If an organism has a reference genome, reads can be directly mapped to it to identify transcripts. Novel transcripts can also be identified, though it is challenging. When a reference genome is not available, the reads can be assembled _de novo_ using tools like `SOAPdenovo-Trans`, `Oases`, `Trans-ABySS`, or `Trinity`. To quantify reads, `HTSeq-count` or `featureCounts` can be used. Quantifications are generally reported in TPM (transcripts per million). To estimate transcript-level expression, `Cufflinks` can be used.

After reads are quantified, differential gene expression analysis can be performed. There are many, many programs that can be used in this effort that the paper exhaustively lists. Another consideration is to analyze alternative splicing either by estimating isoform expression or detecting signals of alternative splicing. Visualization can be performed at the read, processed coverage, unnormalized, or normalized levels. To detect gene fusions, paired-end sequencing should be performed with a large insert size. Small RNAs can also be analyzed, though the sequencing depth tends to be more shallow than standard RNA-seq.

The last step in analysis would be functional annotation. This requires sufficient functional annotation already created by the community, and there are resources like Gene Ontology, Bioconductor, DAVID, or Babelomics that do this. Protein-coding transcripts can be annotated using databases like SwissProt, Pfam, and InterPro.

The final part of the paper discusses integration with other data types. DNA sequencing can be combined for SNIP discovery, RNA-editing analyses, and expression quantitative trait loci mapping. DNA methylation can be combined for epigenetic analysis in relation to RNA-seq. Chromatin features can be combined using ChIP-seq to analyze the effect of a transcription factor on gene expression. MicroRNAs can be combined to understand the regulatory effects of miRNAs on expression levels. Proteomics and metabolomics are controversial to be combined with RNA-seq because of general low correlation but can be used to identify novel isoforms.

The paper concludes with a discussion of what's on the horizon for RNA-seq with advances such as single-cell RNA-seq and long-read sequencing. This paper serves as a high-level primer for how to carry out an RNA-seq experiment, complete with important considerations and plenty of tools and resources that can be used. Any RNA-seq experiment should start with this paper.he paper concludes with a discussion of what's on the horizon for RNA-seq with advances such as single-cell RNA-seq and long-read sequencing. This paper serves as a high level primer for how to carry out a RNA-seq experiment, complete with important considerations and plenty of tools and resources that can be used. Any RNA-seq experiment should start with this paper.


###### _Apologies for the length of the review, this was quite the dense paper!_