**Review of: Current best practices in single‐cell RNA‐seq analysis: a tutorial**

Peter Tran

Much like the popular paper by Mortazavi _et al_. _A survey of best practices for RNA-seq data analysis_, this paper by Luecken and Theis attempts to set the stage for how to perform single-cell RNA-seq (scRNA-seq) analysis. scRNA-seq is filled with hundreds of different tools and workflows, for better or for worse, and the field is difficult for newcomers to enter and become productive. The authors combat that by defining a pipeline. It begins with pre-processing and visualization to obtain molecular counts (count matrices). Quality control must be performed, with three factors: count depth, genes per barcode, and the fraction of counts from mitochondrial genes per barcode. After that, the counts have to be normalized; the most popular of which is counts per million (CPM). Normalization is not enough, data correction must be performed depending on biological or technical factors. After this, feature selection can be used to reduce the dimensionality of the data for more prudent analysis.

Once all the pre-processing steps are done, the meat of the pipeline can be performed, which is the downstream analysis. The first step is to perform clustering by using machine learning to identify clusters of similar cells. Once the clusters are created, they need to be annotated to know what kind of cells are in a cluster; marker genes can be used to do so. After clustering, the authors provide several different routes for analysis: trajectory inference, where the data is interpreted as a snapshot of a continuous process, inferring the transition paths between different cell types; gene expression dynamics, which is using trajectory-associated genes to assist in inferring transition paths highlighted above; metastable stats, where plotting histograms of trajectories highlights different possible transcriptomic states; compositional analysis, which is examining proportions of cells that fall into each cluster, and examining the change in these proportions in particular; and differential expression, not unlike its more basic RNA-seq counterpart, is the measurement of expression levels and their changes between multiple states.

In each of these steps listed above, the authors provide ample detail and references to papers and packages useful. In total, this paper serves as a great primer to someone new to the scRNA-seq field and does a great job of explaining the basics while giving the reader much to consider.
