**Review of: Single-molecule sequencing and chromatin conformation capture enable de novo reference assembly of the domestic goat genome**

Peter Tran

This paper by Smith et al. attempts to push forward current genome assembly techniques to create highly accurate assemblies. They do this by combining several approaches into one: long-read sequencing, short-read sequencing, and optical/chromatin-interaction mapping. With these combined, the authors manage to assemble, _de novo_, a mammalian genome (domestic goat) with very few gaps. With repeats being the biggest hurdle in assembly, chromatin interaction mapping (Hi-C) and optical mapping provide long-range chromosome interaction information, this allows the authors to connect large contigs into scaffolds. To connect small contigs generated from short-read sequencing, the authors use single-molecule sequencing (PacBio) to generate long reads that connect the contigs.
    
The rest of the paper goes into detail the quality of the assembly of the domestic goat genome the authors performed and compares it to the reference genome of the time; which does show remarkable improvement. While long-read sequencing has been helping resolve repeats in assembly for a long time, the authors show that with optical and chromatin interaction mapping, the quality of assembly can be much improved with little cost. The authors estimate $100,000 to complete a similar genome assembly, and while that is three times greater than the standard short-read assembly commonly performed, this comes with much greater quality.