**Review of: Transcriptomics technologies**

Peter Tran

Shafee _et al_. set out to review approaches in transcriptomics in this review paper. The authors start with the history of transcriptomics, wherein the pre-transcriptomics era, early researchers using cDNA with reverse transcriptase with Sanger sequencing to sequence individual transcripts (ESTs). Early attempts involved concatenating random transcript fragments and then sequencing them with Sanger sequencing (SAGE). The contemporary techniques we are used to--microarrays and RNA-seq--were developed in the mid-1990s and 2000s. Microarrays work by placing probes, which are short nucleotide oligos, on a glass assay where random transcripts are poured onto the cell and hybridization with probes will emit fluorescence that can be used to determine what sequence has hybridized. RNA-seq works by using high-throughput sequencing directly on a sample of transcripts, which those reads are then either aligned to a genome or transcriptome or assembled _de novo_ to determine their function. The paper has a detailed methods section for each of these techniques and how to perform them in a wet-lab.

Once data is generated, it needs to be processed. For microarrays, this is rather simple as you just need to process the image of the cell to determine which well is lit and by how much to determine what transcripts you can detect. RNA-seq also has similar image processing where the individual nucleotides can be identified. However, RNA-seq has much more computational work afterward. The pipeline is as follows, you quality control the reads, align them to a reference genome or transcriptome, quantify the alignment, measure differential expression levels between multiple samples, and finally validate your analysis. In conclusion, the authors list several ways transcriptomics techniques can be applied, such as in diagnostics and disease profiling, human and pathogen transcriptomes, responses to environment, gene function annotation, and noncoding RNA.

While this paper was certainly decent and a good introduction to transcriptomics, it pays particular attention to RNA-seq but does not give it the depth of coverage that [Mortazavi _et al_.](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-0881-8) gave in their review paper.
